docs/content/guide/modules.md

```
@@ -97,6 +97,41 @@ export default defineComponent({
 
 See [Quill modules docs](https://quilljs.com/docs/modules/) for more details.
 
+## Registering Formats And Blots
+
+Use a shorthand name for ordinary modules, or pass a full Quill registration path when a package exposes formats or blots that must be registered before the module.
+
+~~~ js
```

markdownlint-cli2 (0.22.1)

[warning] Code fence style
Expected: backtick; Actual: tilde

(MD048, code-fence-style)